app/services/participants/base.rb

```diff
@@ -31,6 +31,8 @@ def call
     def initialize(params:)
       params.each do |param, value|
         send("#{param}=", value)
+      rescue NoMethodError
```

Comment:
Does this leave us open to some kind of injection? Are there any parameters that could be modified which shouldn't be? Picking out a specific list of parameters would be a more obviously safe way of doing it, but so long as we're confident about this...